vm/rust/src/lib.rs

```diff
@@ -80,9 +93,9 @@ pub extern "C" fn cairoVMCall(
     let mut state = CachedState::new(reader);
     let mut resources = ExecutionResources::default();
     let mut context = EntryPointExecutionContext::new(
-        build_block_context(chain_id_str, block_number, block_timestamp),
+        build_block_context(chain_id_str, block_number, block_timestamp, StarkFelt::default()),
         AccountTransactionContext::default(),
-        MAX_STEPS_PER_TX as u32,
+        4_000_000,
```

Comment:
Is this an arbitrary number?

Reply:
got it from the main branch

https://github.com/starkware-libs/blockifier/blob/main/crates/blockifier/src/abi/constants.rs#L29